crates/aptos-telemetry-service/src/context.rs

```diff
@@ -12,7 +13,7 @@ use aptos_types::{chain_id::ChainId, PeerId};
 use jsonwebtoken::{Algorithm, DecodingKey, EncodingKey, TokenData, Validation};
 use serde::{de::DeserializeOwned, Serialize};
 use std::{
-    collections::{HashMap, HashSet},
+    collections::{HashMap, HashSet, VecDeque},
```

Comment:
nit: Why VecDeque instead of a Vec?

Reply:
Makes it more efficient to pop elements from front of the queue.